lib/signs_ui_web/ensure_signs_ui_group.ex

```diff
@@ -0,0 +1,25 @@
+defmodule SignsUiWeb.EnsureSignsUiGroup do
+  import Plug.Conn
+
+  @signs_ui_group "signs-ui-admin"
+
+  @unauthorized_message "You are not authorized to access the signs viewer. " <>
```

Comment:
@lemald I'm not wild about having this big blob of HTML in a module that does authorization logic, is there perhaps an idiomatic way in Phoenix to render a template from within a random plug like this?

Reply:
You can raise an exception, and then have ErrorView render a response for it.

Reply:
I'm not sure if `ErrorView` is quit the correct place to put it (I don't want it to trigger the stacktrace thing-y in development mode), but I did create an actual view and such so it's no longer just a constant in this module.